### PR TITLE
Feat:"Added Product Details Page"

### LINKED ISSUE #7: feat: Product-Details Page

### PR DESCRIPTION
**Pull request by sadathshariff**
Added Product Details Page, on Click it will route and display the Product Info

### CODE REVIEW

File: src/components/Card/ProductCard/ProductCard.jsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
+import { Link } from "react-router-dom";
+import { useFilter } from "../../../contexts/filterContext/context";
```
Comment (rdshinde) [inline src/components/Card/ProductCard/ProductCard.jsx]: I think you can add `index.js` in contexts folder for better management of import and export.

**Review by rdshinde — APPROVED** [review]
Everything looks fine to me.

File: src/components/Card/ProductCard/ProductCard.jsx [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 import styles from "./ProductCard.module.css";
 export const ProductCard = (props) => {
   const { img, name, price, description } = props;
```
Comment (coldpigli) [inline src/components/Card/ProductCard/ProductCard.jsx:5]: could have de-structured in the signature itself.
If you've done it for better readability, then it's fine

File: src/components/ProductDetails/ProductDetail.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.product-container {
+  margin: 1rem auto;
+  width: 60%;
+  display: flex;
```
Comment (coldpigli) [inline src/components/ProductDetails/ProductDetail.css]: could be extracted to a generic class

**Review by coldpigli — APPROVED** [review]
Overall everything looks clean and modular. Suggested some minor changes.

File: src/components/Card/ProductCard/ProductCard.jsx (hunk 2 above)
Comment (sadathshariff) [reply src/components/Card/ProductCard/ProductCard.jsx:5]: Yeah I'm passing the remaining props to Product Details page so did that way

File: src/components/ProductDetails/ProductDetail.css (hunk 3 above)
Comment (sadathshariff) [reply src/components/ProductDetails/ProductDetail.css]: Yeah will do that

**Pull request merged**